docs/learn/learn-nft-pallets.md

```diff
@@ -12,61 +12,127 @@ NFT-related actions within their dApp.
+    collection's configuration is valid (i.e. all options within
+    `config: PalletNftsCollectionConfig`).
+  - [other mint settings](https://github.com/paritytech/substrate/pull/12483) include:
+    - wave minting, for example mint X number of items that go to collection owners and Y number of
+      items for the public
+    - force mint: minting bypassing mint settings
 
-- Buying an item up for sale.
-- Creating (i.e., mint) and burning (i.e., destroy) a collection of items or a single item (burning
-  must be signed either by the admin of the collection or the owner). Creating an item usually
-  involves setting some attributes specific to that item. Creating a collection also requires the
-  specification of the maximum number of items.
-
-  [Different settings](https://github.com/paritytech/substrate/pull/12483) are available for minting
-  collections:
-
-  - public mints: everyone can mint an asset (good for soul-bound collections)
-  - external mint: mint for holders of NFTs in another collection
-  - wave minting: mint X number of items that go to collection owners and Y number of items for the
-    public
-  - force mint: minting bypassing mint settings
-  - change max supply until it gets locked: possibility to change the supply for a limited amount of
-    times
+:::info
+
+Metadata, attributes, and settings of an item or collection can be locked. The user can decide what
+to lock. Also, unauthorized and/or unprivileged transfers can be prevented by locking mechanism
+(unprivileged actions can be re-allowed anytime).
+
+:::
+
+With all these options, one can decide to modify the price of the collection's items and who can
+mint, receive or buy items in that collection. Time constraints are available with `startBlock` and
+`endBlock` parameters. It is thus possible, for example, to create a schedule in which holders of
+items in collection A (`HolderOf` parameter) will be able to claim for free a limited number of NFTs
+from Collection X (`maxSupply` parameter) only within a specific time frame. You can modify the
+parameters so anyone can buy more NFTs from Collection X.
+
+This can be useful for events such as Hackathons where participants who bought a ticket receive the
+NFT ticket from Collection A. Then, all holders of at least one item in Collection A (i.e. all
+ticket holders) will be given free avatar NFT from Collection X within the event schedule. After the
+event, any additional remaining items in Collection X can be made available to the public through a
+marketplace.
+
+The requirement to get the free avatar is being a holder of at least one NFT in Collection A. One
+can only claim the avatar specifying which NFT (i.e. the ID) owns in Collection A. The same NFT
+cannot be used twice, meaning that holders of multiple NFTs in Collection A (for example,
+participants to multiple Hackathons) can claim multiple avatars specific to each event.
```

Comment:
I made a few corrections. It is mentioned that same NFT cannot be used twice. But if someone has multiple NFTs, they should be able to claim multiple avatars right?

```suggestion
The requirement to get the free avatar is to hold at least one NFT in Collection A. One
can only claim the avatar specifying which NFT (i.e. the ID) they own in Collection A. The same NFT
cannot be used twice. Holders of multiple NFTs in Collection A (for example,
participants in multiple Hackathons) can claim multiple avatars specific to each event.
```